test/lib/ansible_test/_internal/host_profiles.py

```diff
@@ -599,12 +1205,12 @@ def wait_until_ready(self) -> str:
             try:
                 return self.get_working_directory()
             except SubprocessError as ex:
-                if 'Permission denied' in ex.message:
-                    raise
-
+                # No "Permission denied" check is performed here.
```

Comment:
It seems like timeouts still happen: https://github.com/ansible/ansible/pull/79508

Reply:
Well, at least we know it's not due to ansible-test trying to connect before the user is ready. It looks like we're back to needing to look over the logs on the guest when it's disconnecting the client.